dspace-server-webapp/src/main/java/org/dspace/app/rest/BitstreamRestController.java

```diff
@@ -241,4 +264,87 @@ public BitstreamResource updateBitstreamFormat(@PathVariable UUID uuid,
         return (BitstreamResource) utils.getResourceRepository(BitstreamRest.CATEGORY, BitstreamRest.NAME)
                 .wrapResource(converter.fromModel(context.reloadEntity(bitstream)));
     }
+
+    /**
+     * This method gets the bundle of the bitstream that corresponds to to the provided bitstream uuid. When multiple
+     * bundles are present, only the first will be returned.
+     *
+     * @param uuid     The UUID of the bitstream for which the bundle will be retrieved
+     * @param response The response object
+     * @param request  The request object
+     * @return The wrapped resource containing the first bundle of the bitstream
+     * @throws IOException
+     * @throws SQLException
+     * @throws AuthorizeException
+     */
+    @PreAuthorize("hasPermission(#uuid, 'BITSTREAM', 'READ')")
+    @RequestMapping(method = {RequestMethod.GET, RequestMethod.HEAD}, value = "/bundle")
+    public ResponseEntity<ResourceSupport> getBundle(@PathVariable UUID uuid, HttpServletResponse response,
```

Comment:
A part of me wonders if these `/bundle` methods should move to a different controller named `BitstreamBundleController`, as the Javadocs of the `BitstreamRestController` state the the controller is "to provide access to the bitstream binary content" (and bundles don't fall under that purpose).  At a minimum, we need to update the JavaDocs here to change the scope of this Controller.

In general, I like the idea of having smaller, more descriptive controllers overall, rather than a generically named controller....as it makes it easier to clarify the scope of what is implemented in the Controller vs what is provided by the `RestRepository` classes.

However, that said, I'm OK with this change for now (if JavaDocs are updated).  We can always refactor later if deemed necessary.